lib/replicator.js

```diff
@@ -2004,6 +2017,10 @@ module.exports = class Replicator {
 
   _ondata (peer, req, data) {
     if (data.block !== null) {
+      this.stats.blocksDownloaded++
+      this.stats.bytesDownloaded += data.block.value.byteLength
```

Comment:
Note: this will crash if a block ever does not contain a Buffer `block.value` entry. The code assumes this never happens (which I think is true, but good to confirm)